### ISSUE TITLE
Can't open repository folder in terminal

### ISSUE THREAD

**Issue by jmejia32** [shiftkey/desktop#167]
## Version

* GitHub Desktop: 2.1.0-linux1

* Operating system: Arch Linux x64

## Steps to Reproduce

1. Open a local repository
2. Go to Repository menu
3. Click on 'Open in `GNOME`(?) Terminal' menu item (I use Deepin DE btw, no GNOME installed)

### Expected Behavior

System's default Virtual Terminal Emulator (Deepin Terminal, in my case) started on repository folder

### Actual Behavior

This message:

![image](https://user-images.githubusercontent.com/15973840/60763870-828bda80-a042-11e9-9949-5aac342735c0.png)

## Additional Information

1. Package installed directly from [AUR](http://aur.archlinux.org/packages/github-desktop-bin)
2. This error may be caused by this issue

![image](https://user-images.githubusercontent.com/15973840/60763810-7c492e80-a041-11e9-88ec-d36113c44e1e.png)

#127 reports same problem but it's related with snap packages and obviously, it's not my case.

### Logs

Log files doesn't see anything related with extrange behavior explained here.

**Comment by immackay** [comment]
Intriguing. I'll investigate.

**Comment by immackay** [comment]
The currently supports shells are Gnome, Mate, Tilix, Terminator, Urxvt, Konsole, Xterm, and Terminology.

I'll open a pull request to provide support for deepin-terminal but it may be a week or two before you see it in a release. Once the PR is merged you can alternatively install from source using [github-desktop-git](https://aur.archlinux.org/packages/github-desktop-git)

**Comment by jmejia32** [comment]
@immackay Great. Thanks, I really appreciate it.

**Comment by nicekiwi** [comment]
I've created a PR based on @immackay 's PR to support the XFCE Terminal too. #331

**Comment by jmejia32** [comment]
Guys, I think we have a problem...

Deepin terminal finally appears in shell menu
![image](https://user-images.githubusercontent.com/15973840/93229532-4ec73d00-f73c-11ea-8d5f-d3360744819e.png)

But It can't open the project folder in terminal yet:
![image](https://user-images.githubusercontent.com/15973840/93229710-7c13eb00-f73c-11ea-994d-5631180ee1d7.png)

![image](https://user-images.githubusercontent.com/15973840/93229160-e4ae9800-f73b-11ea-9458-dd24739a3730.png)

Version: 2.5.4-linux1

**Comment by shiftkey** [comment]
@jmejia32 please open a new issue for that specific terminal, as it might be mis-configured
